webapp/src/main/resources/application.yml

```diff
@@ -89,6 +89,10 @@ permissionservice:
   endpoint: ${rdw.permission.service}
 
 app:
+  reporting:
+    webapp:
+      url: /
```

Comment:
Oh, just saw your comment to JT about navigating to home in the current app by default.